package.json

```diff
@@ -41,10 +41,12 @@
     "ember-cli-moment-shim": "0.6.1",
     "ember-cli-qunit": "1.0.1",
     "ember-cli-uglify": "^1.0.1",
+    "ember-cli-version-checker": "^1.1.4",
```

Comment:
For this to be usable in the blueprint, it needs to be included in `dependencies` not `devDependencies`.